merkle/compact_merkle_tree_test.go

```diff
@@ -92,8 +92,8 @@ func TestAddingLeaves(t *testing.T) {
 			if got, want := tree.CurrentRoot(), roots[i]; !bytes.Equal(got, want) {
 				t.Errorf("CurrentRoot()=%v, want %v", got, want)
 			}
-			if got, want := tree.Hashes(), hasheses[i]; !reflect.DeepEqual(got, want) {
-				t.Errorf("Hashes()=%v, want %v", got, want)
+			if diff := pretty.Compare(tree.Hashes(), hashes[i]); diff != "" {
+				t.Errorf("post-Hashes() diff:\n%v", diff)
```

Comment:
Can we make these "post-Hashes" errors + below indicate which part of the code they come from.